docs/docs-beta/docs/getting-started/concepts.md

```diff
@@ -0,0 +1,641 @@
+---
+title: "Concepts"
+sidebar_position: 200
+---
+
+```mermaid
+%%{
+  init: {
+    'theme': 'base',
+    'themeVariables': {
+      'primaryColor': '#4F43DD',
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#231F1B',
+      'lineColor': '#DEDDFF',
+      'secondaryColor': '#BDBAB7',
+      'tertiaryColor': '#BDBAB7'
+    }
+  }
+
+}%%
+  graph TD
```

Comment:
Will reorganize this graph in a better order when we have sign off on this being the final list of concepts and correct relationships